### ISSUE TITLE
app: naming of parameters to create_app_factory

### ISSUE THREAD

**Issue by lnielsen** [inveniosoftware/invenio-base#32]
See comments in #31 

```
conf_loader
bp_entry_points
ext_entry_points
```

**Comment by lnielsen** [comment]
Comment from #42:

Improve naming convention of `ext_entry_points` and `bp_entry_points` (see https://github.com/inveniosoftware/invenio-base/blob/master/invenio_base/app.py#L42-L43)

- [ ] rename arguments
- [ ] update cookiecutter templates

--
cc @lnielsen

**Comment by lnielsen** [comment]
Before doing any changes, let's agree on naming:

```
conf_loader -> config_loader
bp_entry_points -> blueprint_entry_points
ext_entry_points -> extension_entry_points
```

I.e. singular (blueprint) + plural (entry_points)

**Comment by jirikuncar** [comment]
> ```
conf_loader -> config_loader
bp_entry_points -> blueprint_entry_points
ext_entry_points -> extension_entry_points
```

@lnielsen These names look reasonable. @SamiHiltunen you can start working on it.

**Comment by SamiHiltunen** [comment]
Fixes in pr #43
